src/test/groovy/unit/GatlingPluginTest.groovy

```diff
@@ -78,10 +79,9 @@ class GatlingPluginTest extends GatlingUnitSpec {
 
     def "should create processGatlingResources task"() {
         expect:
-        with(project.tasks.getByName("processGatlingResources")) {
-            it instanceof ProcessResources
-            it != null
-            it.actions.find { it.action instanceof LogbackConfigTaskAction } != null
+        with(project.tasks.getByName("processGatlingResources")) { Task task ->
```

Comment:
@slandelle the actual reason was a clash between implicit closure parameters, aka `it`.
Seems it changed with Gradle update, since it may use different Groovy version.

So, the solution was just to properly declarae explicit closure parameters.
But I also simplified the code in general.

Reply:
> the actual reason was a clash between implicit closure parameters, aka it.

Wow, changing the scope of closure parameters like like a huge breaking change to me. Scary.

`hasProperty` was what I was missing :)

Reply:
When "aspects" are added to task via doLast \ doFirst - they are wrapper in action wrapper class, that's why there always were some "pure" and some "wrapper" tasks.

But, I believe due to Groovy version change, the bug just appread cleary.

Reply:
> Wow, changing the scope of closure parameters like like a huge breaking change to me. Scary.

Scary? it only changes a variable name.